markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -164,7 +163,23 @@ library PerpsAccount {
         Data storage self,
         PerpsPrice.Tolerance stalenessTolerance
     ) internal view returns (bool isEligible, int256 availableMargin) {
-        availableMargin = getAvailableMargin(self, stalenessTolerance);
+        // calculate keeper costs
+        KeeperCosts.Data storage keeperCosts = KeeperCosts.load();
+        uint256 totalLiquidationCost = keeperCosts.getFlagKeeperCosts(self.id) +
```

Comment:
Since this check is relevant for margin only liquidations, should these costs correspond to margin only liquidations?

E.g. maybe this function can use:

```solidity
uint256 keeperReward = Margin.getMarginLiquidationOnlyReward(
            marginValues.collateralUsd,
            PerpMarketConfiguration.load(marketId),
            globalConfig,
            addresses
        );
 ```
 
 Instead

Reply:
yeah this function (`isEligibleForMarginLiquidation`) is only used for margin liquidations.  yes can create a separate function but i'd imagine non blocking for now